### DISCUSSION TITLE
LabWC review and streams in Russian

### DISCUSSION THREAD

**Discussion by glow12121** [labwc/labwc#895] [General]
I thought someone may be interested. I am not the author. You could use automatically translated subtitles if necessary.

A review:

https://www.youtube.com/watch?v=Y1TlAbSd92w

And two streams:

https://www.youtube.com/watch?v=M-culcm7ql0

https://www.youtube.com/watch?v=4JdozTMvz5E

**Comment by johanmalm** [comment]
Thanks for sharing.

**Comment by Consolatis** [comment]
Indeed, thanks for sharing. Sadly the auto translations are not the best, especially in the beginning I didn't quite understand the issue with "the white border". There was also something about not being able to hide the decorations that I didn't understand (maybe it is something about client side decorations?). I would appreciate a sum up of the issues brought up by somebody speaking Russian.

The zoom-in on the rectangle hover effect sticking out over the rounded corner was pretty telling and presented in a funny way.
That one shouldn't be that hard to fix actually, we'd just need to create 2 more rounded corners with their sizes being border_size smaller than the existing one and use those images for the left and right button hover effects instead of the generic rectangular one.

**Comment by glow12121** [comment]
https://www.youtube.com/watch?v=Y1TlAbSd92w

* Overall, Openbox was successfully moved to Wayland, however there are some rough edges.
* Unlike Openbox, you can't determine window sizes.
* Unlike Openbox, you can't remove window decorations (the title bar).
* Round corners problems: only possible for title bars
* It's a matter of taste but round corners on all sides, shadows and blur would be nice.
* XML config is inconvenient. In Openbox, some enthusiasts use SSHKD?? plugin (could be a GUI settings manager, or XML converter).
* No desktop icons is a minus.
* Developers likely won't move away from XML format but if they complete decorations and menu settings, and add desktop icons, that alone will make LabWC a top tier compositor.
* Client-side title bar drawing (analog of GTK's shadows) is a good idea but could be better implemented.
* Sticky window corners is a great extension.
* No workspace and language switchers in stock Waybar out of AUR. It's likely devs won't create workspace switching functionality.
* No problems with Arch installation from AUR.
* First step -- copying the config (Translator: IMO the exact command line could be added in the manual because many users don't know where the default config is.)
* Setting up LabWC is quite simple, although the steps aren't always obvious.
* Had to use a trick to turn off the left click desktop menu.
* Not sure what's the difference between `themerc` and `themerc-override`.
* The project has decent, although at times confusing, documentation.
* Problem with a rounded corner shadow.
* Despite the issues, the compositor is very much usable. The author has been doing all kinds of work on it for a few days. If you're looking for a Wayland compositor, give LabWC a try. 
* Hope the video was useful. Peace.

**Reply by Consolatis** [reply]
Wow, I wasn't expecting a complete transcript here. Thank you very much for the translation!

**Reply by glow12121** [reply]
No problem, it's more of an outline.
